package.json

```diff
@@ -2,12 +2,11 @@
   "name": "sendbird-uikit",
   "version": "2.6.0",
   "description": "React based UI kit for sendbird",
-  "main": "release/index.js",
-  "style": "release/dist/index.css",
-  "typings": "release/index.d.ts",
-  "module": "release/index.js",
+  "main": "dist/cjs/index.js",
```

Comment:
This fixes the jest issue.

I also moved the files into dist. I'm not sure why there was `release` and `dist` but it didn't seem necessary and was easy to clean up.

This should be backwards compatible since users shouldn't be referencing the `release` paths directly.

Reply:
That was due to some legacy customer request, but, can remove now :)